ert_shared/ensemble_evaluator/prefect_ensemble/client.py

```diff
@@ -20,22 +20,17 @@ def __init__(self, url, max_retries=10, timeout_multiplier=5):
         self.websocket = None
         self.loop = asyncio.new_event_loop()
 
-    async def get_websocket(self):
-        return await websockets.connect(self.url)
-
-    async def _send(self, msg):
+    def send(self, msg):
         for retry in range(self._max_retries):
             try:
                 if self.websocket is None:
-                    self.websocket = await self.get_websocket()
-                await self.websocket.send(msg)
+                    self.websocket = self.loop.run_until_complete(websockets.connect(self.url, loop=self.loop))
+                self.loop.run_until_complete(self.websocket.send(msg))
```

Comment:
Isn't this affecting master also? Maybe do this in another branch, since this may take a while to land.